src/main/java/uk/co/ramp/covid/simulation/Model.java

```diff
@@ -229,6 +229,12 @@ private void writeOutput(int iterId, List<List<DailyStats>> s) {
        
     }
 
+    public String modelToJsonString() {
+        Writer writer = new StringWriter();
+        ParameterIO.getGson().toJson(this, writer);
+        return writer.toString();
+    }
+
     private void outputModelParams(Path outF) {
```

Comment:
I haven't combined these into a single function that just takes a Writer since when I tried it didn't add a whole amount (and made things a bit longer/harder to see the flow - it's a few lines each so not a big issue I don't think).